packages/address/src/address.ts

```diff
@@ -44,7 +48,8 @@ export default class Address extends AbstractAddress {
 
   /**
    * Returns the `bech32Address` property
-   *
+   * @deprecated
+   * The `bech32Address` is now deprecated. Please migrate to B256 format (see https://docs.fuel.network/docs/specs/abi/argument-encoding/#b256 for more details) as this will be the standard going forward.
```

Comment:
# For Functions / Methods

```suggestion
```suggestion
   * Type `Bech32Address` is now deprecated, as is this method. Use B256 addresses instead. ([help](https://docs.fuel.network/docs/specs/abi/argument-encoding/#b256))
```

Reply:
8967ed703